contrib/version-up/version.lisp

```diff
@@ -3,7 +3,7 @@
 ;;;don't edit above
 
 (defun path (target)
-  (let ((path (ql:where-is-system target)))
+  (let ((path (uiop:symbol-call :quicklisp 'where-is-system target)))
```

Comment:
Use `'#:where-is-system` instead to avoid interning a new symbol in current package. Or use a keyword.